projects/fhi-angular-highcharts/src/lib/fhi-angular-highcharts.component.html

```diff
@@ -1,8 +1,8 @@
-<article *ngIf="diagramOptions !== undefined">
-  <h1 class="h3 mb-3">{{ diagramOptions.title }}</h1>
+<article *ngIf="diagramOptionsInternal !== undefined">
+  <h1 class="h3 mb-3">{{ diagramOptionsInternal.title }}</h1>
 
-  @if (diagramOptions.description) {
-    <p class="mb-4">{{ diagramOptions.description }}</p>
+  @if (diagramOptionsInternal.description) {
```

Comment:
@proand A combination of *ngIf and @if in one file is a bit untidy.

Reply:
Created a new issue to handle this: #768